specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -5394,6 +5493,57 @@
         }
       }
     },
+    "LineOfCredit": {
+      "description": "Line of credit resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "A line of credit.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/LineOfCreditProperties"
+        }
+      }
+    },
+    "LineOfCreditProperties": {
+      "description": "The properties of the line of credit.",
+      "properties": {
+        "creditLimit": {
+          "description": "The current credit limit.",
+          "$ref": "#/definitions/Amount",
+          "readOnly": true
+        },
+        "reason": {
+          "description": "The reason for the line of credit status when not approved.",
+          "type": "string",
+          "readOnly": true
+        },
+        "remainingBalance": {
+          "description": "Remaining balance.",
+          "$ref": "#/definitions/Amount",
+          "readOnly": true
+        },
+        "status": {
```

Comment:
Should this be an Enum? Declaring the expected values will make this API more consumable.

Reply:
Sure, changed to enum.